resources/js/Directives/getinvolvedDirective.js

```diff
@@ -31,6 +31,21 @@ app.directive('getinvolved', ['$http', function ($http) {
             $scope.loadMore = function () {
                 $scope.totalDisplayed += 20;
             };
+
+            $scope.counter = function () {
+                if(self.contributors) {
+                    var keys = Object.keys(self.contributors);
+                    return keys.length;
+                }
+                return 0;
+            };
+
+            $scope.nomoreItems = function () {
+                if($scope.totalDisplayed > $scope.counter ) {
+                    return 1;
+                }
+                return 0;
```

Comment:
```js
return $scope.totalDisplayed > self.contributors.length
```

but check for `self.contributors` first.

Reply:
the ng-hide & ng-show functions here depend on 0 & 1, 

the earlier 0 & 1 can be removed though (used for counter)